tests/longevity-encryption-at-rest-200GB-6h.yaml

```diff
@@ -27,6 +27,9 @@ authorizer: 'CassandraAuthorizer'
 pre_create_schema: True
 scylla_encryption_options: "{ 'cipher_algorithm' : 'AES/ECB/PKCS5Padding', 'secret_key_strength' : 128, 'key_provider': 'LocalFileSystemKeyProviderFactory', 'secret_key_file': '/tmp/secret_key'}"
 
+# enable system_info_encryption, config kmip_hosts
+append_conf: "system_info_encryption:\n    enabled: True  # system_info_encryption\n    key_provider: LocalFileSystemKeyProviderFactory  # system_info_encryption\n    secret_key_file: '/tmp/system_info_encryption_keyfile'\n\nkmip_hosts:\n     kmip_test:\n         hosts: kmip-interop1.cryptsoft.com\n         certificate: /etc/scylla/conf/SCYLLADB.pem\n         keyfile: /etc/scylla/conf/SCYLLADB.pem\n         truststore: /etc/scylla/conf/CA.pem\n         priority_string: 'SECURE128:+RSA:-VERS-TLS1.0:-ECDHE-ECDSA'\n"
```

Comment:
can we convert this parameter into a real yaml ? or at least to multi line strings, this not that readable

Reply:
I'm using existing append_conf interface, multi line strings doesn't work with it in my testing.

One solution is assigning a file path in sct yaml, and config_setup() will read the file content and append it to scylla.yaml. But it introduces multiple configure files for one job/test.

eg: append_conf_file: /tmp/encryption_conf.yaml

I will investigate yaml-multiline and other solution, thanks

Reply:
@amoskong what you did is very "hacky", 
1. please use YAML format
2. find a way not to use append conf interface.